### DISCUSSION TITLE
Windows 11 Install

### DISCUSSION THREAD

**Discussion by danielmayfield** [spacedriveapp/spacedrive#1992] [Community Support]
## Problem
When trying to install, I get this error.
![image](https://github.com/spacedriveapp/spacedrive/assets/44943172/5a4135ec-19ab-4b42-80de-ca6b6478045e)

## Windows Info
**Version:** 23H2
**OS Build:** 22631.3085

## Spacedrive version
I have tried the latest 0.2.0 build and the previous 0.1.4 build. Both of these have the same error when trying to install.

## Troubleshooting Steps Already Tried

1. Run as admin
2. Tried Winget package
3. Tried spacedrive website download
4. Move installer to C drive
5. Give user full control of temp folder

**Comment by jihuayu** [comment]
I also got this bug.
You can open the Windows Task Manager, search the Windows Installer program, and kill it. 
After doing it, you can install it normally!

**Comment by utkubakir** [comment]
Fixed on version `0.2.1`, thank you for reporting @jihuayu, @danielmayfield
